Cargo.toml

```diff
@@ -222,3 +222,15 @@ conf-files = [
 
 [package.metadata.spellcheck]
 config = "./scripts/ci/gitlab/spellcheck.toml"
+
+# x86_64-unknown-linux-gnu
+[target.x86_64-unknown-linux-gnu.dependencies]
+parity-util-mem = { version = "0.12", default-features = false, features = ["jemalloc-global"] }
+
+# Apple Silicon chips
+[target.aarch64-apple-darwin.dependencies]
+parity-util-mem = { version = "0.12", default-features = false, features = ["mimalloc-global"] }
+
+# linux/arm64
+[target.aarch64-unknown-linux-gnu.dependencies]
+parity-util-mem = { version = "0.12", default-features = false, features = ["mimalloc-global"] }
```

Comment:
no longer needed on current master

Reply:
There were issues at runtime yes but I started this PR a while back.
I will check again master without those profiles, thanks for the notice.